doc/md/CLI/getting_started.mdx

```diff
@@ -84,5 +84,55 @@ curl -LO https://release.ariga.io/atlas/atlas-windows-amd64-v0.0.1.exe
 Move the atlas binary to a file location on your system PATH.
 
 
+</TabItem>
+</Tabs>
+
```

Comment:
getting started is a kind of a tutorial, so i think you should add a section with docker run to start up the demo db.
i don't think it makes sense people will be hesitant to try actions vs a real db at first.

Reply:
I don't think that is required, some people run binaries, others run docker, if you are here you know what you are doing in terms of running a DB.